src/Cache/Cache.php

```diff
@@ -33,7 +33,7 @@
  *
  * ```
  * Cache::config('shared', [
- *    'className' => Cake\Cache\Engine\ApcuEngine::class,
+ *    'className' => \Cake\Cache\Engine\ApcuEngine::class,
```

Comment:
Users would write the original version though, right?

Reply:
Only in a non-namespaced file it work work.